### PR TITLE
chore: Update color tokens

### PR DESCRIPTION
**Pull request by MarinaRomanova**
## 🧑‍ Changes description
Update color tokens for Spark Theme

## 🤔 Context
Color tokens have changed

### CODE REVIEW

File: spark/src/main/kotlin/com/adevinta/spark/tokens/Color.kt [hunk 1]
```diff
@@ -298,16 +454,13 @@ public fun darkSparkColors(
...
- * @property error The error color is used to indicate errors in components, such as invalid text in
- * a text field.
- * @property onError Color used for text and icons displayed on top of the error color.
- * @property errorContainer The preferred tonal color of error containers.
- * @property onErrorContainer The color (and state variants) that should be used for content on
- * top of [errorContainer].
  * @property outline Subtle color used for boundaries. Outline color role adds contrast for
  * accessibility purposes.
+ * @property outlineHigh Utility color used for boundaries for decorative elements when strong
+ *  * contrast is not required.
```
Comment (SimonMarquis) [inline spark/src/main/kotlin/com/adevinta/spark/tokens/Color.kt:460]:
```suggestion
 * contrast is not required.
```

**Review by SimonMarquis — DISMISSED** [review]
LGTM

File: spark/src/main/kotlin/com/adevinta/spark/tokens/Color.kt [hunk 2]
```diff
@@ -135,30 +187,54 @@ public fun lightSparkColors(
     onSurface: Color = Grey10,
     surfaceVariant: Color = GreyBlue90,
     onSurfaceVariant: Color = GreyBlue30,
+    surfaceInverse: Color = Surfer90,
+    onSurfaceInverse: Color = White,
     surfaceTint: Color = primary,
-    outline: Color = GreyBlue50,
-    outlineVariant: Color = GreyBlue80,
     inversePrimary: Color = Orange80,
     inverseSurface: Color = Grey20,
```
Comment (soulcramer) [inline spark/src/main/kotlin/com/adevinta/spark/tokens/Color.kt:194]: This should be removed then
```suggestion

```

File: spark/src/main/kotlin/com/adevinta/spark/tokens/Color.kt [hunk 3]
```diff
@@ -135,30 +187,54 @@ public fun lightSparkColors(
...
     inverseOnSurface: Color = Grey95,
```
Comment (soulcramer) [inline spark/src/main/kotlin/com/adevinta/spark/tokens/Color.kt:195]: This should be removed then
```suggestion

```

File: spark/src/main/kotlin/com/adevinta/spark/tokens/Color.kt [hunk 4]
```diff
@@ -135,30 +187,54 @@ public fun lightSparkColors(
...
+    outline: Color = GreyBlue50,
+    outlineHigh: Color = Surfer20,
+    overlay: Color = Surfer80.copy(alpha = 0.75f),
```
Comment (soulcramer) [inline spark/src/main/kotlin/com/adevinta/spark/tokens/Color.kt:198]: We can remove this as on Android the is handled by the OS
```suggestion

```

File: spark/src/main/kotlin/com/adevinta/spark/tokens/Color.kt [hunk 5]
```diff
@@ -298,16 +454,13 @@ public fun darkSparkColors(
  * sit on top of other surfaces with [surface] color.
```
Comment (soulcramer) [inline spark/src/main/kotlin/com/adevinta/spark/tokens/Color.kt:454]: We should remove these as well

File: spark/src/main/kotlin/com/adevinta/spark/tokens/Color.kt [hunk 6]
```diff
@@ -381,18 +571,50 @@ public class SparkColors(
...
+        internal set
+    public var onSurfaceInverse: Color by mutableStateOf(onSurfaceInverse, structuralEqualityPolicy())
+        internal set
+    @Deprecated("This property will be removed as it is not part of Spark Token")
     public var surfaceVariant: Color by mutableStateOf(surfaceVariant, structuralEqualityPolicy())
         internal set
+    @Deprecated("This property will be removed as it is not part of Spark Token")
     public var onSurfaceVariant: Color by mutableStateOf(onSurfaceVariant, structuralEqualityPolicy())
         internal set
+    @Deprecated("This property will be removed as it is not part of Spark Token")
```
Comment (soulcramer) [inline spark/src/main/kotlin/com/adevinta/spark/tokens/Color.kt:584]: We can remove the Deprecated on this one as it's necessary for Material

**Review by soulcramer — CHANGES_REQUESTED** [review]

**Review by SimonMarquis — APPROVED** [review]

**Pull request merged**